src/pheval/cli_pheval_utils.py

```diff
@@ -11,19 +14,77 @@
     metavar="FILE",
     help="Path to the semantic similarity profile to be scrambled.",
 )
-def scramble_semsim():
+def scramble_semsim(input: click.Path):
     """scramble_semsim"""
     print("running pheval_utils::scramble_semsim command")
 
 
-@click.command()
+@click.command("scramble-phenopackets")
 @click.option(
-    "--input",
-    "-i",
+    "--phenopacket-dir",
+    "-P",
+    metavar="PATH",
+    required=True,
+    help="Path to phenopackets directory",
+)
+@click.option(
+    "--scramble-factor",
+    "-s",
+    metavar=float,
+    required=True,
+    default=0.5,
+    show_default=True,
+    help="Scramble factor for randomising phenopacket phenotypic profiles.",
+    type=float,
+)
+@click.option(
+    "--output-file-suffix",
+    "-o",
+    metavar="<str>",
+    required=True,
+    help="Suffix to append to output file",
+)
+@click.option(
+    "--output-dir",
+    "-O",
+    metavar="PATH",
+    required=True,
+    help="Path for creation of output directory",
+    default="noisy_phenopackets",
+)
+def scramble_phenopackets_command(
+    phenopacket_dir: click.Path,
+    scramble_factor: float,
+    output_file_suffix: str,
+    output_dir: click.Path,
+):
```

Comment:
I think it would be better that if using the scramble-phenopackets command here for it to have all the click options defined as it is in the [create_noisy_phenopackets.py](https://github.com/monarch-initiative/pheval/blob/b05ae24572c39837b39d6237e559aa9df24e6722/src/pheval/prepare/create_noisy_phenopackets.py#L214). Specifically, where I have specified the option type as 'Path'.

Reply:
I would advocate for a clear separation of CLI methods and their implementation. I think cli_pheval_utils should contain all CLI mappings related to pheval utils, and the other modules (like [create_noisy_phenopackets.py](https://github.com/monarch-initiative/pheval/blob/b05ae24572c39837b39d6237e559aa9df24e6722/src/pheval/prepare/create_noisy_phenopackets.py#L214)) only their implementation. Are you advocating to keeping CLI methods in the same files as their implementation @yaseminbridges? 

We still need to work out the correct options - for example, we need to be able to pass in single phenopackets as well as entire corpora - work in progress.

Reply:
No no, I agree that they should be clearly separated. My point was more that when I define the click command, for each option I am defining the type expected, for example [here](https://github.com/monarch-initiative/pheval/blob/b05ae24572c39837b39d6237e559aa9df24e6722/src/pheval/prepare/create_noisy_phenopackets.py#L221) where I've defined the type as the Path. 

There is already a separate command for creating a single noisy phenopacket [here](https://github.com/monarch-initiative/pheval/blob/b05ae24572c39837b39d6237e559aa9df24e6722/src/pheval/prepare/create_noisy_phenopackets.py#L204) which could also be put into this file